spec/controllers/hyrax/my/collections_controller_spec.rb

```diff
@@ -1,29 +1,22 @@
 describe Hyrax::My::CollectionsController, type: :controller do
   describe "logged in user" do
-    let(:user)  { create(:user) }
-    let(:other) { create(:user) }
-
-    let!(:my_file)              { create(:work, user: user) }
-    let!(:first_collection)     { create(:public_collection, user: user) }
-    let!(:unrelated_collection) { create(:public_collection, user: other) }
-
-    before { sign_in user }
-
     describe "#index" do
-      context "with mulitple pages of collections" do
-        before { 2.times { create(:public_collection, user: user) } }
-        it "paginates" do
-          get :index, params: { per_page: 2 }
-          expect(assigns[:document_list].length).to eq 2
-          get :index, params: { per_page: 2, page: 2 }
-          expect(assigns[:document_list].length).to be >= 1
-        end
-      end
+      let(:user)  { create(:user) }
+      let(:other) { create(:user) }
+      before { sign_in user }
+
+      let!(:my_file)              { create(:work, user: user) }
```

Comment:
One of the recent recommendations that rubocop has (that we haven't upgraded to yet) is that we should not name things that we are not referencing and that they should just go into a before block.  I'd prefer we follow that guideline so that we eventually have a simpler upgrade.